botocore/utils.py

```diff
@@ -897,6 +973,14 @@ def deep_merge(base, extra):
         base[key] = extra[key]
 
 
+def get_service_event_name(service_id):
```

Comment:
-1 on this being an external method.
+1 on this being on the service model more directly, perhaps even making a `ServiceID ` object that has blessed transformations like this. I could see this being transformed in other ways too.